### ISSUE TITLE
Release v4.2.38

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/openfoodnetwork#10540]
## Preparation on Thursday

- [x] Merge pull requests in the [Ready To Go] column
- [x] Include translations: `tx pull --force`
- [x] [Draft new release]. Look at previous [releases] for inspiration.
- [x] Notify [#instance-managers] of user-facing changes.

## Testing

- [x] [Find build] of the release commit and copy it below.
- [x] Move this issue to Test Ready.
- [x] Notify `@testers` in [#testing].
- [x] Test build: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master/builds/4244

## Finish on Tuesday

- [x] Publish and notify [#global-community] (this is automatically posted with a plugin)
- [x] Deploy the new release to all managed instances.
  <details><summary>Command line instructions</summary>
  <pre>
  cd ofn-install
  git pull
  ansible-playbook --limit all-prod --extra-vars "git_version=vx.y.z" playbooks/deploy.yml
  </pre>
  </details>
- [x] Notify [#instance-managers]:
  > @instance_managers The new release has been deployed.
- [x] Nudge next release manager

The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.

[Ready To Go]: #zenhub
[Transifex pull request]: https://github.com/openfoodfoundation/openfoodnetwork/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+head%3Atransifex
[Draft new release]: https://github.com/openfoodfoundation/openfoodnetwork/releases/new?tag=v&title=v+Code+Name&body=Congrats%0A%0ADescription%0A%0A%23%23+User+facing+changes+:eyes:%0A%0A%0A%0A%23%23+Technical+changes+:wrench:%0A%0A
[releases]: https://github.com/openfoodfoundation/openfoodnetwork/releases
[#instance-managers]: https://app.slack.com/client/T02G54U79/CG7NJ966B
[#testing]: https://openfoodnetwork.slack.com/app_redirect?channel=C02TZ6X00
[Find build]: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master
[#global-community]: https://app.slack.com/client/T02G54U79/C59ADD8F2

**Comment by drummer83** [comment]
Hi @mkllnk,

I have tested the release. There were some weird things happening on staging AU:
- I couldn't enter credit card details for back office orders as admin (super admin worked fine), because the input fields were not shown. Could this be related to this bugsnag? https://openfoodnetwork.slack.com/archives/G010VN35QU9/p1678450113089559
- After a while they were shown again (but cut off) and the message said 'error saving payment', but the payment was actually sent for authorization.
- And then it was working again. Really strange.
- At some point emails stopped working and so I decided to switch to staging UK, where everything was working great! :tada: 

Here are the testing notes (without the problems mentioned above): https://openfoodnetwork.slack.com/archives/G010VN35QU9/p1678450113089559

Really strange and frustrating but now I think we are ready to release. :partying_face: 
Will check if staging AU 'recovers'.
